docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,334 @@
+* Start Alluxio on multiple nodes
+* Perform basic tasks via Alluxio Shell
+* **[Bonus]** Run sample Hadoop MapReduce job in Alluxio cluster
+* **[Bonus]** Run sample Apache Spark job in Alluxio cluster
+* Shutdown Alluxio
+
+**[Required]** In this documentation, we will use AWS S3 as the example distributed storage as Alluxio under file system.
+Please make sure you have an [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
+you will be able to use S3 in this guide. If you don't have a S3 account, you can also use an existing HDFS cluster as the under
+file system. The "Set up a distributed storage as UFS" section will be slightly different.
+
+**Note**  This guide is meant for you to quickly start interacting with an Alluxio system, in a distributed environment.
+If you are interested in running a larger scale example which highlights the performance benefits of Alluxio,
+try out the instructions in either of these two whitepapers: [Accelerating on-demand data analytics with
+Alluxio](https://alluxio.com/resources/accelerating-on-demand-data-analytics-with-alluxio),
+[Accelerating data analytics on ceph object storage with Alluxio](https://www.alluxio.com/blog/accelerating-data-analytics-on-ceph-object-storage-with-alluxio).
+
+## Prerequisites
+
+For the following quick start guide, you will need:
+
+* 3-nodes Linux cluster
+* [Java 7 or newer](Java-Setup.html)
+* AWS account and keys
+
+### Setup SSH (Linux)
+
+Please make sure on all the nodes, ssh is enabled.
+
+## Downloading Alluxio
+
+First, [download the Alluxio release](http://www.alluxio.org/download) locally. You can
+download the latest {{site.ALLUXIO_RELEASED_VERSION}} release pre-built for various versions of
+Hadoop from the [Alluxio download page](http://www.alluxio.org/download).
+
+Second, `scp` the release tarball to all the Linux nodes. Note that, feel free to use scriptable way (such as Ansible)
+to distribute the tarball and automate the following process on multiple machines. In this example,
+manually commands are shown and most of them should be executed on all the nodes.
+
+Next, you can unpack the download with the following commands on all the nodes. Your filename may be different
+depending on which pre-built binaries you have downloaded.
+
+```bash
+$ tar -xzf alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}
+```
+
+This will create a directory `alluxio-{{site.ALLUXIO_RELEASED_VERSION}}` with all of the Alluxio
+source files and Java binaries.
+
+## Configuring Alluxio
+
+Before we start Alluxio, we have to configure it. Please ensure all the subsequent commands are run on all the Linux nodes.
+
+In the `${ALLUXIO_HOME}/conf` directory, create the `conf/alluxio-site.properties` configuration
+file from the template.
+
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
```

Comment:
User also needs to modify the `conf/workers` file?

Reply:
I don't think so. Registered worker will be added to `conf/workers` automatically.

Reply:
`conf/workers` is not modified by Alluxio. The admin has to add the hostnames to that file, so the Alluxio scripts know which hosts to start workers on. The same goes for masters.

Reply:
Right. That's my understanding too

Reply:
ping, I don't think this demo can work without the `conf/workers` set up. Could you double check it?

Reply:
It should still work with starting everything independently. But to make it explicit, I've added a `TIPS` section to mention the more scalable way of using `conf/workers`, `copyDir` and `start all`